src/server/protocol.ts

```diff
@@ -852,6 +851,10 @@ namespace ts.server.protocol {
          * Length of the span.
          */
         length: number;
+        /**
+         * Optional for backwards compat, use "2020" for modern LSP-like classifications
+         */
+        format?: ts.SemanticClassificationFormat

```

Comment:
@orta Just want to check that this will also update the `protocol.d.ts` file that we consume. I didn't see the semantic APIs in the `d.ts` when I last check